compose/config.go

```diff
@@ -99,6 +99,18 @@ type Config struct {
 	// UseLegacyErrorFormat controls whether the legacy error format (with `error_debug`, `error_hint`, ...)
 	// should be used or not.
 	UseLegacyErrorFormat bool
+
+	// JWTSkipClientAuth indicates, if client authentication can be skipped, when using jwt as assertion.
+	JWTSkipClientAuth bool
```

Comment:
I think "GrantTypeJWTBearerCXanSkipClientAuth" is longer but explains very well what this does :)

Reply:
+